app/controllers/provider_foreman_controller.rb

```diff
@@ -36,12 +36,12 @@ def show_list
   def new
     assert_privileges("provider_foreman_add_provider")
     @provider_cfgmgmt = ManageIQ::Providers::ConfigurationManager.new
-    @provider_types = ["Ansible Tower", "Foreman"]
+    @provider_types = ["Ansible Tower", ui_lookup(:ui_title => 'foreman')]
```

Comment:
cc @blomquisg 

I think this needs some work, because this needs to happen at the provider level not the manager level.  Unfortuantely, we don't subclass provider into provider types, however, that's sort of intentional because a provider can span multiple types.  As such, we have to go bottom up, I think. So I expect we'd need something like

```ruby
# Get the parent providers for all known configuration managers.
ManageIQ::Providers::ConfigurationManager.supported_subclasses.map {|c| c.parent::Provider}
```

Also the filename makes me :sob:, but alas.